src/client/packaging/pypi/CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 All notable future changes to the `delphi_epidata` python client will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## [4.2.0] - 2023-09-27
```

Comment:
We're not going to 4.2.0 right? The plan is to just go to 4.1.11?

Reply:
i have some text for the changelog that im about to push that addresses this